src/coll/mod.rs

```diff
@@ -546,29 +553,29 @@ impl Collection {
         resolve_options!(self, options, [write_concern]);
 
         let update = Update::new(self.namespace(), query, update.into(), false, options);
-        RUNTIME.block_on(self.client().execute_operation(&update, None))
+        self.client().execute_operation(&update, None).await
     }
 
     /// Creates the indexes specified by `models`.
-    pub fn create_indexes(
+    pub async fn create_indexes(
         &self,
         models: impl IntoIterator<Item = IndexModel>,
     ) -> Result<Vec<String>> {
         unimplemented!()
     }
 
     /// Drops the index specified by `name`.
-    pub fn drop_index(&self, name: &str) -> Result<Document> {
```

Comment:
We said we wanted to get rid of these since they aren't implemented, right? It might make sense to do it now, since we're already making a bunch of other breaking changes here.

Reply:
done